libs/notebook_helpers.py

```diff
@@ -1,32 +1,13 @@
 from typing import Dict
 import logging
-import os
 import pandas as pd
 
-from libs.datasets import dataset_utils
 from libs.datasets.sources import nytimes_dataset
 from libs.datasets.sources import can_scraper_local_dashboard_providers
 
 _logger = logging.getLogger(__name__)
 
 
-def set_covid_data_public():
```

Comment:
This further depreciates some of the notebooks, but the notebooks that use this method don't appear to be functional anyway. if we'd like, I can try to update the notebooks to work with this update, but I'm not sure that's worth the effort

Reply:
Nah, definitely don't bother.  I've never used the notebooks. I'm not sure who did TBH.  I think we could just delete them if they get in our way. 🤷

Reply:
yeah sounds good--it seems like they were used for exploratory purposes--I've never looked at them either haha